defaults/main.yml

```diff
@@ -8,6 +8,7 @@ varnish_default_vcl_template_path: default.vcl.j2
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
 
+varnish_listen_address: ""
```

Comment:
Should this be explicitly 0.0.0.0 by default (or 127.0.0.1 etc)?  Just non blank?